android/build.gradle

```diff
@@ -39,4 +49,30 @@ dependencies {
     implementation 'androidx.coordinatorlayout:coordinatorlayout:1.1.0'
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.0.0'
     implementation 'com.google.android.material:material:1.1.0'
+    implementation "androidx.core:core-ktx:1.6.0"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.5.20"
+}
+
+configurations {
+    ktlint
+}
+
+dependencies {
+    ktlint "com.pinterest:ktlint:0.40.0"
+}
+
+task ktlint(type: JavaExec, group: "verification") {
+    description = "Check Kotlin code style."
```

Comment:
Is it somehow possible to check if there are any kotlin files staged inside the script? For now it fails if you have unstaged changes in kotlin files. If it is not possible, then we can leave as it is now.

Reply:
Fixed in 44e9db403908ac48f5a6379c65074676ebd416cb